Polyfills/Console/Source/Console.cpp

```diff
@@ -16,15 +16,15 @@ namespace Babylon::Polyfills::Internal
                 ParentT::InstanceMethod("log", &Console::Log),
                 ParentT::InstanceMethod("warn", &Console::Warn),
                 ParentT::InstanceMethod("error", &Console::Error),
-            });
+            },
+            &callback);
 
-        Napi::Object console = func.New({Napi::External<Babylon::Polyfills::Console::CallbackT>::New(env, new Babylon::Polyfills::Console::CallbackT(std::move(callback)))});
-        env.Global().Set(JS_INSTANCE_NAME, console);
+        env.Global().Set(JS_INSTANCE_NAME, func.New({}));
     }
 
     Console::Console(const Napi::CallbackInfo& info)
         : ParentT{info}
-        , m_callback{*info[0].As<Napi::External<Babylon::Polyfills::Console::CallbackT>>().Data()}
+        , m_callback{std::move(*(Babylon::Polyfills::Console::CallbackT*)info.Data())}
```

Comment:
nit: This seems like an unusual pattern. I don't think it will cause problems as written, but taking ownership of the callback this way seems a bit confusing to me. Could we instead set the callback member directly from the static function? In other words, change the initializer here to 
```
, m_callback{}
```
and change the calling code to 
```
auto console = func.New({});
Console::Unwrap(console)->m_callback = std::move(callback);
env.Global().Set(JS_INSTANCE_NAME, console);
```
or something like that? Slightly less efficient, but very clear what it's doing.